Makefile

```diff
@@ -242,8 +242,9 @@ clean:
 
 .PHONY: fmt
 fmt:
-	@echo "Running gitea-fmt (with gofumpt)..."
 	@MISSPELL_PACKAGE=$(MISSPELL_PACKAGE) GOFUMPT_PACKAGE=$(GOFUMPT_PACKAGE) $(GO) run build/code-batch-process.go gitea-fmt -w '{file-list}'
+	$(eval TEMPLATES := $(wildcard templates/**/*.tmpl))
+	@$(SED_INPLACE) -e 's/{{[ 	]\{1,\}/{{/g' -e '/^[ 	]\{1,\}}}/! s/[ 	]\{1,\}}}/}}/g' $(TEMPLATES)
```

Comment:
Is it only me, or is the second expression weird?
I can completely understand the first one, but the second seems absolutely wrong:
Why doesn't it start with `s/`?
Why is the `s/` in the middle?
What's the `/! s/`?
And am I right to assume that it is some kind of unnecessary condition?

Reply:
I picked it up from [here](https://stackoverflow.com/a/12178023/808699), it's basically a "ignore pattern" that makes the subsequent replacement not apply.

It is for cases like this:

```handlebars
  {{
      stuff
  }}
```

without it, the `}}` would get dedented, which is not desirable.

Reply:
Added a comment describing it. doing it via `perl` might result in a much more readable regexp, but I'm not sure whether we can depend on it (likely can, thought).

Reply:
I think sed is fine for now. If we are to switch to perl, we should replace all `sed` calls with it.